docs/fsharp/get-started/get-started-vscode.md

```diff
@@ -42,58 +42,50 @@ You must also install the [.NET Core SDK](https://www.microsoft.com/net/download
-
-    ![](media/getting-started-vscode/vscode-ext.png)
+Next, click the Extensions icon and search for "Ionide":
 
 The only plugin required for F# support in Visual Studio Code is [Ionide-fsharp](https://marketplace.visualstudio.com/items?itemName=Ionide.Ionide-fsharp). However, you can also install [Ionide-FAKE](https://marketplace.visualstudio.com/items?itemName=Ionide.Ionide-FAKE) and to get [FAKE](https://fsharp.github.io/FAKE/) support and [Ionide-Paket](https://marketplace.visualstudio.com/items?itemName=Ionide.Ionide-Paket) to get [Paket](https://fsprojects.github.io/Paket/) support. FAKE and Paket are additonal F# community tools for building projects and managing dependencies, respectively.
 
 ## Creating your first project with Ionide
 
 To create a new F# project, open Visual Studio Code in a new folder (you can name it whatever you like).
 
-![](media/getting-started-vscode/vscode-open-dir.png)
+### [Windows](#tab/windows)
 
-Next, open the Command Palette (Ctrl+Shift+P on Windows, ⌘+Shift+P on macOS, Ctrl+Shift+P on Linux) and type the following:
+Next, open the Command Palette (Ctrl+Shift+P) and type the following:
 
-```
->f#: New Project
-```
+>F# new project
 
-This is powered by the [FORGE](https://github.com/fsharp-editing/Forge) project.  You should see something like this:
+### [macOS](#tab/macos)
 
-![](media/getting-started-vscode/vscode-new-proj.png)
+Next, open the Command Palette (⌘+Shift+P) and type the following:
 
-> [!NOTE]
-If you don't see the above, try refreshing templates by running the following command in the Command Palette: `>F#: Refresh Project Templates`.
+>F# new project
 
-Select "F#: New Project" by hitting **Enter**, which will take you to this step:
+### [Linux](#tab/linux)
 
-![](media/getting-started-vscode/vscode-proj-type.png)
+Next, open the Command Palette (Ctrl+Shift+P) and type the following:
 
-This will select a template for a specific type of project.  There are quite a few options here, such as an [FsLab](https://fslab.org) template for Data Science or [Suave](https://suave.io) template for Web Programming.  This article uses the `classlib` template, so highlight that and hit **Enter**.  You will then reach the following step:
+>F# new project
 
-![](media/getting-started-vscode/vscode-new-dir.png)
+---
+
+This is powered by the [FORGE](https://github.com/fsharp-editing/Forge) project. You should see something like this:
 
-This lets you create the project in a different directory, if you like.  Leave it blank for now.  It will create the project in the current directory.  Once you press **Enter**, you will reach the following step:
+> [!NOTE]
+If you don't see template options, try refreshing templates by running the following command in the Command Palette: `>F#: Refresh Project Templates`.
 
-![](media/getting-started-vscode/vscode-proj-name.png)
+Select "F#: New Project" by hitting **Enter**, which will take you to the next step, which allows you to select project types.
```

Comment:
which will take you to the next step, which allows you to select project types -> which takes you to the next step, that allows you to select project types